validate.sh

```diff
@@ -7,9 +7,14 @@
 
 set -eou pipefail
 
-for json in $(ls -1 examples/*); do
-  for schema in $(ls -1 schema/*); do
-    echo "Testing json '$json' against schema '$schema'"
-    ajv test -s $schema -d $json --valid
-  done
+for json in $(ls -1 examples/*-bundle.json); do
```

Comment:
Perhaps consider adding a comment above each loop describing their purpose?  (Suggests the contributor whose track record of helpful comments has been lacking of late 😅 )

Reply:
You could probably refactor this code to be better than mine. I wanted to be more clever, but gave up and went for simple.

Docs added.